src/main/java/core/utils/AmazonUtils.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * @(#) AmazonUtils.java Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package core.utils;
+
+import core.selenium.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import project.ui.BasePage;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * AmazonUtils class.
+ *
+ * @author John Salazar Pinto
+ * @version 1.0
+ */
+public class AmazonUtils {
+    private WebDriver driver;
+    WebDriverManager webDriverManager;
+    private static String url = "https://www.calculadora.org/";
```

Comment:
Calculadora URL should not be used.

Reply:
done

Reply:
Maybe this get () should be outside the web driver manager.